pkg/controller/direct/workstations/config_mappings.go

```diff
@@ -127,8 +127,6 @@ func WorkstationConfigSpec_ToProto(mapCtx *direct.MapContext, in *krm.Workstatio
 	out.ReadinessChecks = direct.Slice_ToProto(mapCtx, in.ReadinessChecks, WorkstationConfig_ReadinessCheck_ToProto)
 	out.ReplicaZones = in.ReplicaZones
 
-	ApplyWorkstationConfigGCPDefaults(mapCtx, in, out, actual)
```

Comment:
This had to be moved out so that the mapping function would satisfy the interface required by the fuzz tests. Applying the defaults requires passing in the actual WorkstationConfig proto.